pkg/graph/schema.graphql

```diff
@@ -264,6 +264,144 @@ type CreateAccessibilityRequestDocumentPayload {
   userErrors: [UserError!]
 }
 
+"""
+The shape of a solution for a business case
+"""
+type BusinessCaseSolution {
+  acquisitionApproach: String
+  cons: String
+  costSavings: String
+  hasUi: String
+  hostingCloudServiceType: String
+  hostingLocation: String
+  hostingType: String
+  pros: String
+  securityIsApproved: Boolean
+  securityIsBeingReviewed: String
+  summary: String
+  title: String
+}
+
+"""
+enum 
+"""
+enum LifecycleCostPhase {
+  """
+  development
+  """
+  DEVELOPMENT
+  """
+  Operations and Maintentance
+  """
+  OPERATIONS_AND_MAINTENANCE
+  """
+  Other
+  """
+  OTHER
+}
+
+"""
+enum 
+"""
+enum LifecycleCostSolution {
+  """
+  A
+  """
+  A
+  """
+  As is
+  """
+  AS_IS
+  """
+  B
+  """
+  B
+  """
+  Preferred
+  """
+  PREFERRED
+}
+
+"""
+enum 
+"""
+enum LifecycleCostYear {
+  """
+  1
+  """
+  LIFECYCLE_COST_YEAR_1
+  """
+  2
+  """
+  LIFECYCLE_COST_YEAR_2
+  """
+  3
+  """
+  LIFECYCLE_COST_YEAR_3
+  """
+  4
+  """
+  LIFECYCLE_COST_YEAR_4
+  """
+  5
+  """
+  LIFECYCLE_COST_YEAR_5
+}
+
+"""
+The shape of an estimated lifecycle cost row
+"""
+type EstimatedLifecycleCost {
+  businessCaseId: UUID!
+  cost: Int
+  id: UUID!
+  phase: LifecycleCostPhase
+  solution: LifecycleCostSolution
+  year: LifecycleCostYear
+}
+
+"""
+Business Case status options
+"""
+enum BusinessCaseStatus {
+  """
+  closed
+  """
+  CLOSED
+  """
+  open
+  """
+  OPEN
+}
+
+"""
+A Business Case instance
+"""
+type BusinessCase {
+  alternativeASolution: BusinessCaseSolution
```

Comment:
This is a bit of a change from the model itself, but the fields are so repetitive. Hoping this is easier for the frontend to handle, too.